cdk/lib/dashboard.ts

```diff
@@ -13,32 +17,36 @@ export interface CacclMonitoringProps {
 }
 
 export class CacclMonitoring extends Construct {
-  dashboard: Dashboard;
+  dashboard: cloudwatch.Dashboard;
+
+  region: string;
 
   constructor(scope: Construct, id: string, props: CacclMonitoringProps) {
     super(scope, id);
 
     const { stackName } = Stack.of(this);
-    const { region } = Stack.of(this);
 
     const { cacclLoadBalancer, cacclService } = props;
     const { loadBalancer } = cacclLoadBalancer;
     const { ecsService } = cacclService;
 
     const dashboardName = `${stackName}-metrics`;
-    this.dashboard = new Dashboard(this, 'Dashboard', { dashboardName });
+    this.dashboard = new cloudwatch.Dashboard(this, 'Dashboard', {
+      dashboardName,
+    });
```

Comment:
Same thing on "if one's on its own line, all on their own line" but again, that's just me